### ISSUE TITLE
@cssprop type missing from viewer

### ISSUE THREAD

**Issue by juliexiong** [web-padawan/api-viewer-element#29]
I have multiple CSS props with similar naming conventions, which can easily become overwhelming and confusing. I want to further define their modification purposes:

```js
/**
* @cssprop {BackgroundColor} [--card-background-color=white] - Background color of card
* @cssprop {Color} [--card-color=black] - Text/icon color in a card
* @cssprop {BorderColor} [--card-border-color=hotpink] - Border color of card
**/
```

There was a similar [issue](https://github.com/runem/web-component-analyzer/issues/114) for this in web-component-analyzer and a [PR](https://github.com/runem/web-component-analyzer/pull/122) that addressed it as a typehint.

**Comment by web-padawan** [comment]
Thanks, will cover that as well as #26.

**Comment by web-padawan** [comment]
Done by #30 and will be included to the upcoming `0.4.0` release.
